src/main/java/uk/gov/hmcts/reform/civil/model/dq/Applicant1DQ.java

```diff
@@ -1,36 +1,40 @@
 package uk.gov.hmcts.reform.civil.model.dq;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 import uk.gov.hmcts.reform.civil.model.StatementOfTruth;
 import uk.gov.hmcts.reform.civil.model.documents.Document;
 
 @Setter
 @Data
 @Builder(toBuilder = true)
+@NoArgsConstructor
+@AllArgsConstructor
 public class Applicant1DQ implements DQ {
 
-    private final FileDirectionsQuestionnaire applicant1DQFileDirectionsQuestionnaire;
-    private final DisclosureOfElectronicDocuments applicant1DQDisclosureOfElectronicDocuments;
-    private final DisclosureOfNonElectronicDocuments applicant1DQDisclosureOfNonElectronicDocuments;
-    private final DisclosureReport applicant1DQDisclosureReport;
-    private final Experts applicant1DQExperts;
-    private final ExpertDetails applicant1RespondToClaimExperts;
-    private final Witnesses applicant1DQWitnesses;
-    private final Hearing applicant1DQHearing;
-    private final HearingLRspec applicant1DQHearingLRspec;
-    private final SmallClaimHearing applicant1DQSmallClaimHearing;
-    private final Document applicant1DQDraftDirections;
-    private final RequestedCourt applicant1DQRequestedCourt;
-    private final HearingSupport applicant1DQHearingSupport;
-    private final FurtherInformation applicant1DQFurtherInformation;
-    private final WelshLanguageRequirements applicant1DQLanguage;
-    private final StatementOfTruth applicant1DQStatementOfTruth;
-    private final VulnerabilityQuestions applicant1DQVulnerabilityQuestions;
-    private final FutureApplications applicant1DQFutureApplications;
-    private final WelshLanguageRequirements applicant1DQLanguageLRspec;
```

Comment:
whats the purpose for removing final?

Reply:
i added the annotation for noargs and allargs constructors and it doesn't work with final